[core]/es_extended/server/commands.lua

```diff
@@ -132,6 +132,16 @@ end, false, {
 	}
 })
 
+ESX.RegisterCommand('repair', 'admin', function(xPlayer, args, showError)
+	local ped = GetPlayerPed(xPlayer.source)
+	local pedVehicle = GetVehiclePedIsIn(ped, false)
+	if not pedVehicle or GetPedInVehicleSeat(pedVehicle, -1) ~= ped then
+		return
+		showError(TranslateCap('not_in_vehicle'))
```

Comment:
No it's not good, if you return 139 row 140 row not execute. Replace to row 139 to 140 140 to 139.